api/v1/paths/members/{memberId}.js

```diff
@@ -22,6 +22,47 @@ const get = async (req, res) => {
   }
 };
 
+/**
+ * @summary Delete member by unique ID.
+ */
+const del = async (req, res) => {
+  try {
+    const { memberId } = req.params;
+    const result = await membersDao.deleteMember(memberId);
+    if (result.rowsAffected < 1) {
+      errorBuilder(res, 404, 'A member with the specified Id was not found');
+    } else {
+      res.sendStatus(204);
+    }
+  } catch (err) {
+    errorHandler(res, err);
+  }
+};
+
+/**
+ * @summary Patch member by unique id
+ */
+const patch = async (req, res) => {
+  try {
+    const { memberId } = req.params;
+    if (memberId !== req.body.data.id) {
+      errorBuilder(res, 400, ['Member id in path does not match id in the body.']);
+    } else {
+      const result = await membersDao.patchMember(memberId, req.body.data.attributes);
+      if (result.rowsAffected < 1) {
```

Comment:
I think this logic should be handled by the DAO since it pertains to the data source. You could return something like `undefined` to signal the path handler that the member could not be found.